about.pug

```diff
@@ -8,6 +8,17 @@ block prepend title
 
 block content
     h1 About
+    p.
+        Puyo Puyo VS is a client that can connect to internet lobbies in order to play Puyo Puyo Matches against up to 10 people at the same time.
```

Comment:
client feels a bit jargon-y for this page. Can we try to reword this a bit? I'm thinking something like:

```jade
Puyo Puyo VS is a fan game that allows you to connect to online lobbies and play Puyo Puyo matches with up to 10 players at the same time.
```